### PR TITLE
restic quickstart doc for alpha testing

### PR DESCRIPTION
**Pull request by skriss**
Signed-off-by: Steve Kriss <steve@heptio.com>

### CODE REVIEW

File: docs/restic.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Restic Integration
+
+Ark has alpha support for backups/restores of pod volume data using [restic][1], an open-source tool for doing
```
Comment (ncdc) [inline docs/restic.md]: I'd remove
- alpha
- linux

File: docs/restic.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+```
+
+```bash
+# have Ark generate a random encryption key
+ark restic init-repository --namespace YOUR_NAMESPACE --key-size ENCRYPTION_KEY_SIZE
+```
+
+**IMPORTANT**: store this key safely and securely. All restic backup data is encrypted and cannot be accessed
+without this key.
+
```
Comment (ncdc) [inline docs/restic.md]: Add a TODO that we'll be supporting key rotation soon

File: docs/restic.md (hunk 1 above)
Comment (ncdc) [reply docs/restic.md]: And also mention that names/commands/etc may change as we receive feedback and refine things

**Comment by skriss** [comment]
Addressed comments

**Comment by ncdc** [comment]
LGTM and we can iterate as needed post-merge.

**Pull request merged**